src/Classes/ServiceAPI/MyRadio_User.php

```diff
@@ -677,6 +707,10 @@ public function getAccountLocked()
      */
     public function getOfficerships($includeMemberships = false)
     {
+        if ($this->getHideProfile()){
+            return [];
+        }
+
```

Comment:
i think not having these on myradio, instead the public website won't display the pages of people who've got hide profile, cos then you couldn't see it in myradio, or is that what we're going for?

Reply:
I was going for a future proof approach. That way any services we have that get user user data both inside and outside of myradio are affected.

Reply:
👍